src/Processors/QtiV2/In/Interactions/TextEntryInteractionMapper.php

```diff
@@ -3,20 +3,42 @@
 namespace LearnosityQti\Processors\QtiV2\In\Interactions;
 
 use LearnosityQti\Entities\QuestionTypes\clozetext;
+use LearnosityQti\Entities\QuestionTypes\clozetext_response_container;
 use LearnosityQti\Processors\QtiV2\In\Validation\TextEntryInteractionValidationBuilder;
 
 class TextEntryInteractionMapper extends AbstractInteractionMapper
 {
+    const MAX_CHAR_LIMIT_DEFAULT = 250;
```

Comment:
Lets remove the `_DEFAULT` here. I think `MAX_CHAR_LIMIT` is suffice.

Reply:
Hm I actually think the `MAX` is misleading, as there's nothing saying the character limit can't be more. This is just a default value so it probably makes more sense to just say `CHAR_LIMIT_DEFAULT`.